<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolchainOptions</class>
 <widget class="QWidget" name="ToolchainOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure toolchain</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Preprocessing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="subsampleCb">
        <property name="text">
         <string>Run on subsampled image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_bgCb">
        <property name="text">
         <string>Remove background (slow)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="deskewCb">
        <property name="text">
         <string>Deskew</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_noiseCb">
        <property name="text">
         <string>Remove noise</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Binarization method:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bin_algoCbox"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Page segmentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Find separators</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="find_sepsCbox"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
